src/core.js

```diff
@@ -841,8 +841,9 @@
 
   };
 
-  win.xtag = xtag;
-  if (typeof define == 'function' && define.amd) define(xtag);
+  if (typeof define === 'function' && define.amd) define(xtag);
```

Comment:
and, I'm just [moving it](https://github.com/x-tag/core/pull/126/files#diff-7eb52b366866677666470e019283c8eaR846)